view/frontend/web/js/view/cart/shipping-rates.js

```diff
@@ -125,9 +125,9 @@ define(
                         success : function (response)
                         {
                             $('#s_method_sucursal').attr('checked',false);
-                            andreaniSucursalLocalidad.append('<option>Seleccione una localidad</option>');
                             for(var i = 0; i < response.length; i++)
                             {
+                            andreaniSucursalLocalidad.append('<option>Seleccione una localidad</option>');
```

Comment:
Esto fue corregido previamente. Si se ejecuta ese metodo dentro del for se multiplica el option de "Seleccione una localidad"